bondcpp/CMakeLists.txt

```diff
@@ -16,10 +19,14 @@ find_package(rclcpp_lifecycle REQUIRED)
 find_package(smclib REQUIRED)
 find_package(Threads REQUIRED)
 
-find_package(PkgConfig REQUIRED)
-pkg_check_modules(UUID REQUIRED uuid)
-# we have to find the absolute path to uuid as target_link_directories is not available before cmake 3.13
-find_library(uuid_ABS_PATH ${UUID_LIBRARIES} PATHS ${UUID_LIBRARY_DIRS})
+if(UNIX AND NOT APPLE)
```

Comment:
What happens on macOS?  If we are never finding the uuid library, how does it build/link against it?